docs/console/project-configuration/create-a-project.mdx

```diff
@@ -149,7 +149,7 @@ The Project creation process is divided into three phases:
    * **Environment name**: the name given to your environment;
    * **Environment ID**: the human-readable ID set to your environment;
    * **Project Host**: the host which exposes documentation and APIs;
-   * **Backoffice Host**: the host which exposes CMS and backoffice APIs;
+   * **Microfrontend Composer Host**: the host which exposes CMS and Microfrontend Composer APIs;
```

Comment:
Are you sure they changed the label of this variable and that it's now called this?

Reply:
You are right, it was correctly written as "Backoffice"